### PR TITLE
[hotfix][doc] fix doc of chapter etl

### PR DESCRIPTION
**Pull request by nyingping**
## What is the purpose of the change

Fix doc of chapter etl.

## Brief change log

1. Add InputSelectable link address ,add  missing `)` at the end of note.
2. Updated the Chinese definition of ` truly Desperate `. In context, I think `迫切渴望` is better than `感到绝望`.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no)no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no)no
  - The serializers: (yes / no / don't know)no 
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know)no 
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / no / don't know)no 
  - The S3 file system connector: (yes / no / don't know)no

## Documentation

  - Does this pull request introduce a new feature? (yes / no)no
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)not applicable

### CODE REVIEW

**Comment by nyingping** [comment]
@flinkbot run azure

File: docs/content/docs/learn-flink/etl.md [hunk 1]
```diff
@@ -475,7 +475,7 @@ mentioned on the `control` stream, and those words are being filtered out of the
...
+It is important to recognize that you have no control over the order in which the `flatMap1` and `flatMap2` callbacks are called. These two input streams are racing against each other, and the Flink runtime will do what it wants to regarding consuming events from one stream or the other. In cases where timing and/or ordering matter, you may find it necessary to buffer events in managed Flink state until your application is ready to process them. (Note: if you are truly desperate, it is possible to exert some limited control over the order in which a two-input operator consumes its inputs by using a custom Operator that implements the [`InputSelectable`](https://nightlies.apache.org/flink/flink-docs-release-1.15/api/java/org/apache/flink/streaming/api/operators/InputSelectable.html) interface.)
```
Comment (MartijnVisser) [inline docs/content/docs/learn-flink/etl.md:478]:
```suggestion
It is important to recognize that you have no control over the order in which the `flatMap1` and `flatMap2` callbacks are called. These two input streams are racing against each other, and the Flink runtime will do what it wants to regarding consuming events from one stream or the other. In cases where timing and/or ordering matter, you may find it necessary to buffer events in managed Flink state until your application is ready to process them. (Note: if you are truly desperate, it is possible to exert some limited control over the order in which a two-input operator consumes its inputs by using a custom Operator that implements the {{< javadoc name="InputSelectable" file="org/apache/flink/streaming/api/operators/InputSelectable.html" >}}
```

File: docs/content.zh/docs/learn-flink/etl.md [hunk 2]
```diff
@@ -399,7 +399,7 @@ public static class ControlFunction extends RichCoFlatMapFunction<String, String
 
 在 Flink 运行时中，`flatMap1` 和 `flatMap2` 在连接流有新元素到来时被调用 —— 在我们的例子中，`control` 流中的元素会进入 `flatMap1`，`streamOfWords` 中的元素会进入 `flatMap2`。这是由两个流连接的顺序决定的，本例中为 `control.connect(streamOfWords)`。
 
-认识到你没法控制 `flatMap1` 和 `flatMap2` 的调用顺序是很重要的。这两个输入流是相互竞争的关系，Flink 运行时将根据从一个流或另一个流中消费的事件做它要做的。对于需要保证时间和/或顺序的场景，你会发现在 Flink 的管理状态中缓存事件一直到它们能够被处理是必须的。（注意：如果你真的感到绝望，可以使用自定义的算子实现 `InputSelectable` 接口，在两输入算子消费它的输入流时增加一些顺序上的限制。）
+认识到你没法控制 `flatMap1` 和 `flatMap2` 的调用顺序是很重要的。这两个输入流是相互竞争的关系，Flink 运行时将根据从一个流或另一个流中消费的事件做它要做的。对于需要保证时间和/或顺序的场景，你会发现在 Flink 的管理状态中缓存事件一直到它们能够被处理是必须的。（注意：如果你真的迫切需要，可以使用自定义的算子实现 [`InputSelectable`](https://nightlies.apache.org/flink/flink-docs-release-1.15/api/java/org/apache/flink/streaming/api/operators/InputSelectable.html) 接口，在两输入算子消费它的输入流时增加一些顺序上的限制。）
```
Comment (MartijnVisser) [inline docs/content.zh/docs/learn-flink/etl.md:402]:
```suggestion
认识到你没法控制 `flatMap1` 和 `flatMap2` 的调用顺序是很重要的。这两个输入流是相互竞争的关系，Flink 运行时将根据从一个流或另一个流中消费的事件做它要做的。对于需要保证时间和/或顺序的场景，你会发现在 Flink 的管理状态中缓存事件一直到它们能够被处理是必须的。（注意：如果你真的迫切需要，可以使用自定义的算子实现 {{< javadoc name="InputSelectable" file="org/apache/flink/streaming/api/operators/InputSelectable.html" >}} 接口，在两输入算子消费它的输入流时增加一些顺序上的限制。）
```

**Review by MartijnVisser — CHANGES_REQUESTED** [review]
@nyingping Thanks but we have a specific way of linking to JavaDoc, see suggestions.

**Comment by nyingping** [comment]
@MartijnVisser Thanks for review and suggestions!

**Review by MartijnVisser — APPROVED** [review]
@nyingping I've rebased your PR and I'll merge it now. Thanks for the improvement!

**Pull request merged**